dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/config/FlywayConfig.java

```diff
@@ -49,7 +48,7 @@ public class FlywayConfig {
 
   private static final String FLYWAY_MIGRATION_FOLDER = "org/hisp/dhis/db/migration";
 
-  @Bean(value = "flyway", initMethod = "migrate")
+  @Bean(initMethod = "migrate")
   @Profile("!test-h2")
```

Comment:
Will we or can we already remove `@Profile("!test-h2")` here?

Reply:
I also want to get rid of it but want to understand our whole config better first. Make the `WebTestConfiguration` only build on a config from the service integration tests.

(I think it could already work as I define `NoOpFlywayConfig` before `FlywayConfig` in `WebTestConfiguration`.)